pants.toml

```diff
@@ -250,15 +249,16 @@ setupdir = "%(pants_supportdir)s/eslint"
 config = "%(pants_supportdir)s/eslint/.eslintrc"
 ignore = "%(pants_supportdir)s/eslint/.eslintignore"
 
+[flake8]
+config = "build-support/flake8/.flake8"
+extra_requirements.add = ["flake8-pantsbuild>=2.0"]
```

Comment:
2.x is the Python-3 only series of the plugin.

Reply:
Would be good to bound this to the next major version... ie, `<3` to avoid causing issues with our own CI when we release breaking changes, heh.

Reply:
Will forward fix. (I want to remove a couple lints we skip here + add some plugins).